sys/include/xtimer.h

```diff
@@ -184,120 +189,129 @@ static inline void xtimer_tsleep64(xtimer_ticks64_t ticks);
  *
  * This function will spin-block, so only use it *very* short periods.
  *
- * @param[in] ticks  the number of xtimer ticks the thread should spin for
+ * @param[in] ticks   the number of xtimer ticks the thread should spin for
  */
 static inline void xtimer_spin(xtimer_ticks32_t ticks);
 
 /**
  * @brief will cause the calling thread to be suspended until the absolute
- * time (@p last_wakeup + @p period).
+ * time (@p last_wakeup_ticks + @p period_us [in ticks]).
```

Comment:
I think `[in ticks]` is ambiguous (Do I pass a ticks argument or a µs argument?), how about:
```
(@p last_wakeup_ticks + period_ticks). @p period_us will be internally converted to ticks.
```